src/mono/wasm/runtime/dotnet.d.ts

```diff
@@ -139,6 +139,10 @@ type MonoConfig = {
      * application environment
      */
     applicationEnvironment?: string;
+    /**
+     * query string to be used for asset loading
+     */
+    assetUniqueQuery?: string;
```

Comment:
What is the benefit over `WebAssemblyStartOptions.loadBootResource`?

Reply:
We could unify those later. Right now, the `loadBootResource` is not there for non-blazor templates.